### PR TITLE
Add dev container to standardize dev environment

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

To standardize dev environment, it uses vscode remote container.

## Issue reference

#22 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		"ms-python.python"
+	],
+
+	"postCreateCommand": "pip install -r ./tests/requirements.txt",
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+	// Uncomment to connect as a non-root user. See https://aka.ms/vscode-remote/containers/non-root.
+	// "remoteUser": "vscode"
+}
```
Comment (artursouza) [inline .devcontainer/devcontainer.json]: Add mounting points for docker in docker.

Comment (youngbupark) [reply .devcontainer/devcontainer.json]: fixed.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"ms-python.python"
+	],
+	
+	"mounts": [ 
+		"source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
+	],
+
+	"postCreateCommand": "pip install -r ./tests/requirements.txt",
```
Comment (yaron2) [inline .devcontainer/devcontainer.json]: where is this file taken from?

Comment (youngbupark) [reply .devcontainer/devcontainer.json]: tests/requirements.txt already exists in the repo. this pr is just to add dev-container.

**Pull request merged**